README.md

```diff
@@ -67,36 +67,34 @@ ide.json.path=subfolder/from/root
 file.json.name=newFileName.json
 ```
 
-**Step 1:** Install `RegisterCompanion` onto the device where you want to mock In-app Billing. You can find the latest version in the [Releases Tab](https://github.com/nytm/Register/releases).
-
-**Step 2:** Add Register as a dependency to your client app:
+**Step 1:** Add Register as a dependency to your client app, this will install the companion app:
 ```groovy 
-compile 'com.nytimes.android:register:0.0.3'
+compile 'com.nytimes.android:register:0.0.8'
 ```
 
 or, on Android Gradle Plugin 3.0 or later:
 
 ```groovy 
-implementation 'com.nytimes.android:register:0.0.3'
+implementation 'com.nytimes.android:register:0.0.8'
 ```
 
-**Step 3:** Create a test Google Services provider (or a real provider):
+**Step 2:** Create a test Google Billing Client (or a real client):
 
 ```java
- private void initGoogleServiceProvider() {
-        if (prefsManager.isUsingTestGoogleServiceProvider()) {
-            googleServiceProvider = new GoogleServiceProviderTesting();
+ private void initGooglePlayBillingClient() {
+        if (prefsManager.isUsingTestBillingClient()) {
+            googleBillingClient = new BillingClientTesting(...);
         } else {
-            googleServiceProvider = new GoogleServiceProviderImpl();
+            googleBillingClient = new BillingClient(...);
```

Comment:
This should also be `GoogleServiceProvider`'s builder

Reply:
True, but in that case we should provide a builder, no?